Add missing gifs for examples on the home page

jwunderl opened the pull request:
As peli pointed out in https://github.com/Microsoft/pxt-arcade/commit/1acdda690598e7a2823f70d15e752c89d63d28c5, there is no duck.gif.

pelikhan commented:
The build automatically picks the available .gif references by the markdown page. Can we get a duck.gif? :)

jwunderl commented:
Sure, I'll scan through and see what all is missing gifs real quick then (I think space destroyer?)

pelikhan commented on docs/examples/space-destroyer.md:
```diff
@@ -42,6 +42,7 @@ ship.setFlag(SpriteFlag.StayInScreen, true)
 ship.bottom = 120
 controller.moveSprite(ship, 100, 100)
 info.setLife(3)
+effects.starField.startScreenEffect()
```
How long does a screen effect last by default?

jwunderl replied:
Forever, and it takes precedence over other effects so it shouldn't ever turn off

jwunderl commented:
Okay, added gifs for the rest of the blocks-games. The javascript games carousel looked fine with pngs for the most part, just added a gif for the bouncer bucket. Good to merge whenever I think

pelikhan commented:
Can you produce PRs in the repo itself?

jwunderl commented:
I'm not a collaborator on this repo quite yet, just core / common packages / microbit
